src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceActions.tsx

```diff
@@ -118,12 +120,14 @@ const NetworkInterfaceActions: FC<NetworkInterfaceActionsProps> = ({
   return (
     <>
```

Comment:
u can remove the fragment, there is no meaning for it

Reply:
Done!